src/fenwick.jl

```diff
@@ -82,15 +97,52 @@ julia> prefixsum(f, 3)
 ```
 """
 function prefixsum(ft::FenwickTree{T}, ind::Integer) where T
+    n = length(ft)
+    @boundscheck 1 <= ind <= n || throw(ArgumentError("$ind should be between 1 and $n"))
     sum = zero(T)
-    i = ind
-    n = ft.n
-    @boundscheck 1 <= i <= n || throw(ArgumentError("$i should be in between 1 and $n"))
-    @inbounds while i > 0
-        sum += ft.bi_tree[i]
-        i -= i&(-i)
+    @inbounds while ind > 0
+        sum += ft.bi_tree[ind]
+        ind -= ind&(-ind)
     end
-    sum
+    return sum
+end
+
+"""
+    suffixsum(sft::SuffixFenwickTree{T}, ind)
+    
+Return the cumulative sum from `ind` upto length of the [`SuffixFenwickTree`](@ref)
+
+# Examples
+```
+julia> f = SuffixFenwickTree{Int}(6)
+julia> inc!(f, 2, 5)
+julia> suffixsum(f, 1)
+ 5
+julia> suffixsum(f, 3)
+ 0
+```
+"""
+function suffixsum(sft::SuffixFenwickTree{T}, ind::Integer) where T
+    n = length(sft)
+    @boundscheck 1 <= ind <= n || throw(ArgumentError("$ind should be between 1 and $n"))
+    sum = zero(T)
+    @inbounds while ind <= n
+        sum += sft.bi_tree[ind]
+        ind += ind&(-ind)
+    end
+    return sum
 end
 
 getindex(ft::FenwickTree{T}, ind::Integer) where T = prefixsum(ft, ind)
+getindex(sft::SuffixFenwickTree{T}, ind::Integer) where T = suffixsum(sft, ind)
+
+function resize!(sft::SuffixFenwickTree{T}, size::Int) where T
+    @boundscheck size > 0 || throw(ArgumentError("size should be greater than 0"))
+    n0 = length(sft)
+    resize!(sft.bi_tree, size)
+    z = zero(T)
+    @inbounds for i = n0+1: size
```

Comment:
preferred style: not using `=` in for loops
```suggestion
    @inbounds for in n0+1 : size
```